webviz_subsurface/_private_plugins/tornado_plot.py

```diff
@@ -220,6 +233,10 @@ def _calc_tornado(reference, scale, cutbyref, data):
                     reference=reference,
                     scale=scale,
                     cutbyref=cutbyref,
+                    number_format=number_format,
+                    unit=unit,
+                    spaced=spaced,
+                    locked_si_prefix=locked_si_prefix,
```

Comment:
Can we do the `data.get(...)` calls directly here instead of defining them above the `try`?